packages/core/src/components/Modal/ModalHeader/ModalHeader.styled.tsx

```diff
@@ -5,7 +5,27 @@ import { Props } from './types';
 
 export const CloseButton = styled(Button)`
     padding: 0;
-    margin-right: -24px;
+    margin-right: -36px;
+    margin-top: -58px;
+
+    svg {
+        border-radius: 50%;
+        padding: 4px;
+        background-color: ${({ theme }) => theme.colors.grey[100]};
+
+        > path {
+            fill: ${({ theme }) => theme.colors.black} !important;
```

Comment:
@gmukul01 , i need help removing this `!important` tag and the one below on `line 26`. 
I can't find where the fill that is overriding this is coming from.